mlrun/api/utils/projects/follower.py

```diff
@@ -346,11 +346,18 @@ def _sync_projects(self, full_sync=False):
                         "Found project in the DB that is not in leader. Removing",
                         name=project_to_remove,
                     )
-                    mlrun.api.crud.Projects().delete_project(
-                        db_session,
-                        project_to_remove,
-                        mlrun.common.schemas.DeletionStrategy.cascading,
-                    )
+                    try:
+                        mlrun.api.crud.Projects().delete_project(
+                            db_session,
+                            project_to_remove,
+                            mlrun.common.schemas.DeletionStrategy.cascading,
+                        )
+                    except Exception as exc:
+                        logger.warning(
+                            "Failed to delete project from DB, continuing...",
+                            name=project_to_remove,
+                            exc=exc,
```

Comment:
use mlrun.errors.err_to_str it logs the exception nicer
```suggestion
                            exc=err_to_str(exc),
```